### PR TITLE
doc: Clarify language specific figures format in docs

### PR DESCRIPTION
**Pull request by tmetzl**
Subject: Clarify language specific figure format in docs

### Feature or Bugfix
- Bugfix (Clarification)

### Purpose
- make it clearer how to name language specific figures

### Detail
- format is given as figure_language_filename, should be figure.language.filename
- add example of how to name a figure

### CODE REVIEW

File: doc/usage/configuration.rst [hunk 1]
```diff
@@ -660,10 +660,11 @@ documentation on :ref:`intl` for details.
    generated by Sphinx will be in that language.  Also, Sphinx will try to
    substitute individual paragraphs from your documents with the translation
    sets obtained from :confval:`locale_dirs`.  Sphinx will search
-   language-specific figures named by `figure_language_filename` and substitute
-   them for original figures.  In the LaTeX builder, a suitable language will
-   be selected as an option for the *Babel* package.  Default is ``None``,
-   which means that no translation will be done.
+   language-specific figures named by `figure.language.filename` 
+   (e.g. the German version of `myfigure.png` will be `myfigure.de.png`)
```
Comment (tk0miya) [inline doc/usage/configuration.rst]: Looks better. But ``` `` ``` (double back-ticks) are better for a literal string in reStructuredText. I think the original text `figure_language_filename` is also not good. Could you use it instead of a single one?

**Review by tk0miya — CHANGES_REQUESTED** [review]

File: doc/usage/configuration.rst (hunk 1 above)
Comment (tmetzl) [reply doc/usage/configuration.rst]: Changed it for the filename example. Should I also change it for the figure.language.filename?

Comment (tk0miya) [reply doc/usage/configuration.rst]: I should be fixed also. But I'll do it on another PR.

**Review by tk0miya — APPROVED** [review]

**Comment by tk0miya** [comment]
Merged. Thank you for your contribution!

File: doc/usage/configuration.rst [hunk 2]
```diff
@@ -660,10 +660,11 @@ documentation on :ref:`intl` for details.
    generated by Sphinx will be in that language.  Also, Sphinx will try to
    substitute individual paragraphs from your documents with the translation
    sets obtained from :confval:`locale_dirs`.  Sphinx will search
-   language-specific figures named by `figure_language_filename` and substitute
-   them for original figures.  In the LaTeX builder, a suitable language will
-   be selected as an option for the *Babel* package.  Default is ``None``,
-   which means that no translation will be done.
+   language-specific figures named by `figure.language.filename`
```
Comment (tk0miya) [inline doc/usage/configuration.rst:663]: Hmm? Why did you change `figure_language_filename` to `figure.language.filename`? This is a configuration variable. So this should not be changed.

File: doc/usage/configuration.rst [hunk 3]
```diff
@@ -660,10 +660,11 @@ documentation on :ref:`intl` for details.
    generated by Sphinx will be in that language.  Also, Sphinx will try to
    substitute individual paragraphs from your documents with the translation
    sets obtained from :confval:`locale_dirs`.  Sphinx will search
-   language-specific figures named by `figure_language_filename` and substitute
-   them for original figures.  In the LaTeX builder, a suitable language will
-   be selected as an option for the *Babel* package.  Default is ``None``,
-   which means that no translation will be done.
+   language-specific figures named by `figure.language.filename` 
+   (e.g. the German version of ``myfigure.png`` will be ``myfigure.de.png``)
```
Comment (tk0miya) [inline doc/usage/configuration.rst:664]: Note: this filename is generated by the default setting of `figure_language_filename`. So we should add a phrase like "by default setting".

File: doc/usage/configuration.rst (hunk 2 above)
Comment (tmetzl) [reply doc/usage/configuration.rst:663]: Oh I mistook this for an example. I did not know it was a config variable.

Comment (tk0miya) [reply doc/usage/configuration.rst:663]: Okay. I fixed it in another PR.
